assets/src/features/home-page/components/HomePageVideoTile.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import MediaControlButton from "../../../pages/room/components/MediaControlButton";
+import { activeButtonStyle, neutralButtonStyle } from "../../../pages/room/components/MediaControlButtons";
+import { TrackWithId } from "../../../pages/room/components/StreamPlayer/MediaPlayerPeersSection";
+import MediaPlayerTile from "../../../pages/room/components/StreamPlayer/MediaPlayerTile";
+import { AUDIO_TRACKS_CONFIG, VIDEO_TRACKS_CONFIG } from "../../../pages/room/consts";
+import { useMedia } from "../../../pages/room/hooks/useMedia";
+import { usePeersState } from "../../../pages/room/hooks/usePeerState";
+import { useSetLocalUserTrack } from "../../../pages/room/hooks/useSetLocalUserTrack";
+import { remoteTrackToLocalTrack } from "../../../pages/room/VideochatSection";
+import InitialsImage, { computeInitials } from "../../room-page/components/InitialsImage";
+import Camera from "../../room-page/icons/Camera";
+import CameraOff from "../../room-page/icons/CameraOff";
+import Microphone from "../../room-page/icons/Microphone";
+import MicrophoneOff from "../../room-page/icons/MicrophoneOff";
+import { usePreviewSettings } from "../hooks/usePreviewSettings";
+
+type HomePageVideoTileProps = {
+  displayName: string;
+};
+
+const HomePageVideoTile: React.FC<HomePageVideoTileProps> = ({ displayName }) => {
+  const { cameraAutostart, audioAutostart } = usePreviewSettings();
+  const { state: peerState, api: peerApi } = usePeersState();
+
+  const localPeer = peerState.local;
+
+  const videoTrack: TrackWithId | null = remoteTrackToLocalTrack(localPeer?.tracks["camera"]);
+  const audioTrack: TrackWithId | null = remoteTrackToLocalTrack(localPeer?.tracks["audio"]);
+  const initials = computeInitials(displayName);
+
+  const localCamera = useMedia(VIDEO_TRACKS_CONFIG, cameraAutostart.status);
+  useSetLocalUserTrack("camera", peerApi, localCamera.stream, localCamera.isEnabled);
+
+  const localAudio = useMedia(AUDIO_TRACKS_CONFIG, audioAutostart.status);
+  useSetLocalUserTrack("audio", peerApi, localAudio.stream, localAudio.isEnabled);
+
+  return (
+    <MediaPlayerTile
+      key="room-preview"
+      peerId={localPeer?.id}
+      video={videoTrack ?? undefined}
```

Comment:
Passing `undefined` here is not a great practice. Especially if the video is not an additional prop. Please change the props of the `MediaPlayerTile` so it has `null` as `nothing`.